asistente_ladm_col/gui/qgis_model_baker/dlg_import_schema.py

```diff
@@ -284,6 +290,12 @@ def update_configuration(self):
         configuration.create_import_tid = CREATE_IMPORT_TID
         configuration.stroke_arcs = STROKE_ARCS
 
+        # Set java path
+        settings = QSettings()
+        java_path = settings.value('QgisModelBaker/ili2db/JavaPath', '', str)
```

Comment:
Can we use `get_java_path_from_qgis_model_baker()`?

Reply:
Done